docs/ux-flow-map.md

```
@@ -0,0 +1,438 @@
+  J --> M[Delete product]
+
+  classDef green fill:#dcfce7,stroke:#16a34a,color:#14532d;
+  classDef yellow fill:#fef9c3,stroke:#ca8a04,color:#713f12;
+  classDef orange fill:#ffedd5,stroke:#ea580c,color:#7c2d12;
+```
+
+### Ambiguities
+
+| Severity | Finding | Evidence / location | Suggested direction |
+| --- | --- | --- | --- |
+| 🟠 | Selecting a project can auto-open the first chat document or first file. This is surprising and hides where the user is. | `handleProjectSelect` opens `firstChat || first document`. | Show a product home/overview page first, with clear cards: Chat, Files, Artifacts, Workflows. |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 4; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)